in-person-payments-example/Options/AdyenOptions.cs

```diff
@@ -24,7 +24,12 @@ public class AdyenOptions
         /// <summary>
         /// Your unique ID for the POS system (cash register) to send this request from.
         /// </summary>
-        public string ADYEN_POS_SALE_ID { get; set; } 
+        public string ADYEN_POS_SALE_ID { get; set; }
 
+        /// <summary>
+        /// Default: null, unless you want to override this to point to a different endpoint based on your region.
```

Comment:
Maybe in the comment mentioned that can be also used to set the Mock Terminal API